### DISCUSSION TITLE
Restrict geohashes length

### DISCUSSION THREAD

**Discussion by ptneves** [filipecorrea/vicinityhash#30] [Q&A] [answered]
Is it possible to restrict the number of geohashes returned by `.convert()`?

**Comment by filipecorrea** [comment, marked as answer]
There's no way of specifying the exact number, but you can reduce it by configuring `precision`:

```typescript
const config = {
    precision: 4
  }

const geohashes = vicinityhash.convert(geofence, config)
```
